### PR TITLE
produce parsed covid care maps csv

### PR DESCRIPTION
**Pull request by TomGoBravo**
## This PR

* Copies logic of https://github.com/covid-projections/covid-data-model/blob/master/libs/datasets/sources/covid_care_map.py to this repo
* Splits missing and extra field warnings in rename_fields. Adds parameter `check_extra_fields` to disable extra fields check because the covid care map has lots of big fields that aren't worth listing in the Field enum.
* Fixes problem in `test/update_covid_county_data_test.py` where the test fails when the window is big

## Tested

Ran `python ./scripts/update_covid_care_map.py --no-fetch` and then, with another WIP covid-data-model PR, was able to reproduce the existing combined datasets.

### CODE REVIEW

File: scripts/helpers.py [hunk 1]
```diff
@@ -16,22 +17,26 @@ def load_county_fips_data(fips_csv: pathlib.Path) -> pd.DataFrame:
     return df
 
 
-def rename_fields(df, fields, already_transformed_fields, log) -> pd.DataFrame:
+def rename_fields(
+    df, fields, already_transformed_fields, log, *, check_extra_fields=True,
+) -> pd.DataFrame:
     """Return df with columns renamed according to fields, logging and dropping unexpected columns."""
```
Comment (ghop02) [inline scripts/helpers.py:23]: Can you expand this docstring?

File: scripts/update_covid_care_map.py [hunk 2]
```diff
@@ -49,7 +91,59 @@ def update(self):
...
+
+        all_df = all_df.set_index([CommonFields.FIPS], verify_integrity=True)
+
+        all_df[CommonFields.COUNTRY] = "USA"
+
+        # Override Washoe County ICU capacity with actual numbers.
+        all_df.at["32031", CommonFields.ICU_BEDS] = 162
+
+        # Overriding NV ICU capacity numbers with actuals
+        all_df.at[STATE_TO_FIPS["NV"], CommonFields.ICU_BEDS] = 844
```
Comment (ghop02) [inline scripts/update_covid_care_map.py:123]: We should add in the modified utah value as well

Comment (ghop02) [reply scripts/update_covid_care_map.py:123]: ah it's not being updated.  I would still prefer to revert the csv change here: https://github.com/covid-projections/covid-data-public/pull/152/commits/be79d85e1aaa058f4c5d0a472c84047b59d7f3d8 and surface the value change in this code

**Review by ghop02 — APPROVED** [review]
Would like to surface the UT modification here if possible, but otherwise LGTM

File: scripts/update_covid_care_map.py (hunk 2 above)
Comment (TomGoBravo) [reply scripts/update_covid_care_map.py:123]: I added a patch for UT ICU_BEDS. When I ran the fetch a bunch of values changed which I didn't feel like testing so going to leave the patch and move on for today.

File: scripts/helpers.py (hunk 1 above)
Comment (TomGoBravo) [reply scripts/helpers.py:23]: added types and more in docstring

**Pull request merged**